docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
 
   # PostgreSQL
   postgres-95:
-    image: camptocamp/postgis:9.5
+    image: sushantdhiman/postgres:9.5
```

Comment:
New image with all extensions we need i.e. `btree`, `postgis` and `hstore`

Reply:
:+1: